yarn.lock

```diff
@@ -1920,16 +1920,14 @@
   dependencies:
     "@inline-svg-unique-id/react" "^1.2.3"
 
-"@manuscripts/body-editor@1.6.2":
+"@manuscripts/body-editor@../manuscripts-body-editor":
```

Comment:
Presumably this will be changed before testing so I'm just pointing to it so it won't be forgotten.